feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionDayTab.kt

```diff
@@ -38,7 +41,11 @@ internal fun SessionDayTab(
     Tab(
         selected = selected,
         onClick = { onTabClicked(index) },
-        modifier = Modifier.height(height).padding(end = 8.dp).clip(CircleShape)
+        modifier = Modifier.height(height).padding(end = 8.dp).clip(CircleShape).semantics {
+            if (day == Day2) {
+                testTag = "sessionDayTab"
```

Comment:
Thanks!
I think this makes code change a little difficult. Day2 is not only day tab.
I think we don't have to switch tab at the moment. What do you think?

Reply:
I'm sorry. I am late in noticing your comment 🙇 
I didn't understand about other influences.. 
So, remove the `semantics` movement and try again 👍